commit 449c6741b69cb745214f1b62e188852f28c79332
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Thu Sep 23 16:28:53 2010 -0700

    Staging: brcm80211: clean up makefile cflag lines
    
    Change to use the proper ccflags-y option, as well as splitting the
    options out one-per-line so that we can see what is needed to be cleaned
    up and removed over time.
    
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Cc: jason <jason@lakedaemon.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/Makefile
+++ b/drivers/staging/brcm80211/Makefile
@@ -15,7 +15,23 @@
 # OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 # CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-EXTRA_CFLAGS := -DBCMDBG -DWLC_HIGH -DSTA -DWME -DWL11N -DDBAND -DBCMDMA32 -DBCMNVRAMR -Idrivers/staging/brcm80211/sys -Idrivers/staging/brcm80211/phy -Idrivers/staging/brcm80211/util -Idrivers/staging/brcm80211/include -Wall -Wstrict-prototypes -Werror -Wmissing-prototypes
+ccflags-y :=					\
+	-DBCMDBG				\
+	-DWLC_HIGH				\
+	-DSTA					\
+	-DWME					\
+	-DWL11N					\
+	-DDBAND					\
+	-DBCMDMA32				\
+	-DBCMNVRAMR				\
+	-Idrivers/staging/brcm80211/sys		\
+	-Idrivers/staging/brcm80211/phy		\
+	-Idrivers/staging/brcm80211/util	\
+	-Idrivers/staging/brcm80211/include	\
+	-Wall					\
+	-Wstrict-prototypes			\
+	-Werror					\
+	-Wmissing-prototypes
 
 PCI_CFLAGS := -DWLC_LOW
 
@@ -59,7 +75,7 @@ obj-m	+= $(MODULEPFX).o
 
 # PCI driver
 ifeq ($(CONFIG_BRCM80211_PCI),y)
-EXTRA_CFLAGS += $(PCI_CFLAGS)
+ccflags-y += $(PCI_CFLAGS)
 $(MODULEPFX)-objs	= $(BRCM80211_OFILES) $(PCIFILES)
 endif
 
--- a/drivers/staging/brcm80211/brcmfmac/Makefile
+++ b/drivers/staging/brcm80211/brcmfmac/Makefile
@@ -15,16 +15,43 @@
 # OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 # CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-DHDCFLAGS = -DLINUX -DBCMDRIVER -DBCMDONGLEHOST -DDHDTHREAD -DBCMWPA2	\
-	-DUNRELEASEDCHIP -Dlinux -DDHD_SDALIGN=64 -DMAX_HDR_READ=64	\
-	-DDHD_FIRSTREAD=64 -DDHD_GPL -DDHD_SCHED -DBDC -DTOE -DDHD_BCMEVENTS	\
-	-DSHOW_EVENTS -DBCMSDIO -DDHD_GPL -DBCMLXSDMMC -DBCMPLATFORM_BUS	\
-	-Wall -Wstrict-prototypes -Werror -DOEM_CHROMIUMOS -DEMBEDDED_PLATFORM	\
-	-DARP_OFFLOAD_SUPPORT -DPKT_FILTER_SUPPORT -DBRCM_FULLMAC	\
-	-DCONFIG_CFG80211 -DMMC_SDIO_ABORT -DDHD_DEBUG_TRAP -DBCMDBG -DDHD_DEBUG	\
-	-Idrivers/staging/brcm80211/brcmfmac	\
+ccflags-y :=			\
+	-DLINUX			\
+	-DBCMDRIVER		\
+	-DBCMDONGLEHOST		\
+	-DDHDTHREAD		\
+	-DBCMWPA2		\
+	-DUNRELEASEDCHIP	\
+	-Dlinux			\
+	-DDHD_SDALIGN=64	\
+	-DMAX_HDR_READ=64	\
+	-DDHD_FIRSTREAD=64	\
+	-DDHD_GPL		\
+	-DDHD_SCHED		\
+	-DBDC			\
+	-DTOE			\
+	-DDHD_BCMEVENTS		\
+	-DSHOW_EVENTS		\
+	-DBCMSDIO		\
+	-DDHD_GPL		\
+	-DBCMLXSDMMC		\
+	-DBCMPLATFORM_BUS	\
+	-Wall			\
+	-Wstrict-prototypes	\
+	-Werror			\
+	-DOEM_CHROMIUMOS	\
+	-DEMBEDDED_PLATFORM	\
+	-DARP_OFFLOAD_SUPPORT	\
+	-DPKT_FILTER_SUPPORT	\
+	-DBRCM_FULLMAC		\
+	-DCONFIG_CFG80211	\
+	-DMMC_SDIO_ABORT	\
+	-DDHD_DEBUG_TRAP	\
+	-DBCMDBG		\
+	-DDHD_DEBUG		\
+	-Idrivers/staging/brcm80211/brcmfmac		\
 	-Idrivers/staging/brcm80211/brcmfmac/include	\
-	-Idrivers/staging/brcm80211/include	\
+	-Idrivers/staging/brcm80211/include		\
 	-Idrivers/staging/brcm80211/util
 
 DHDOFILES = dhd_linux.o linux_osl.o bcmutils.o dhd_common.o dhd_custom_gpio.o	\
@@ -34,5 +61,4 @@ DHDOFILES = dhd_linux.o linux_osl.o bcmu
 
 obj-$(CONFIG_BRCMFMAC) += brcmfmac.o
 brcmfmac-objs += $(DHDOFILES)
-EXTRA_CFLAGS = $(DHDCFLAGS)
 EXTRA_LDFLAGS += --strip-debug
